smallrye-reactive-messaging-http/src/main/java/io/smallrye/reactive/messaging/http/JsonHelper.java

```diff
@@ -26,6 +26,14 @@ public static JsonObject asJsonObject(Config config) {
                 // Ignore me
             }
 
+            try {
+                boolean b = config.getValue(key, Boolean.class);
```

Comment:
This returns "false" for whatever is not "true". 
That explains the next branch where we compare it with "false" (line 39)

Reply:
Yes, I thought so, judging from the failed test. I am not that familiar with the MP Config API, so I would expect this to return empty in case the value is not a boolean. Turns out my intuition about this API was wrong (too?), because I did not read about "Automatic Converters".

So, how to proceed? Would the following work?

```java
Object o = config.getValue(key, Object.class);
if (o instanceof Boolean) {
  ...
}
else if (o instanceof Integer) {
  ...
}
else if (o instanceof String) {
  ...
}
...
```

Also, feel free to close the PR if you like.